src/sdk.ts

```diff
@@ -705,10 +705,9 @@ export class OpenSeaSDK {
 
       // If the order is using offer protection, the extraData
       // must be included with the order to successfully fulfill.
-      const fulfillmentOrder =
-        result.fulfillment_data.transaction.input_data.orders[0];
-      if ("extraData" in fulfillmentOrder) {
-        extraData = (fulfillmentOrder as AdvancedOrder).extraData;
+      const inputData = result.fulfillment_data.transaction.input_data;
```

Comment:
for my understanding, line 708 should be: 

`const fulfillmentOrder = result.fulfillment_data.orders[0];`

Can you recheck it? btw, I have no idea how to test the new fix on my side. It looks like you modified ts file and it should be built it locally. I do not know how to using the modified version on my side. Thanks

Reply:
i believe the orders object doesn't contain the AdvancedOrder extraData, so we have to look at input_data. you can try modifying the changed lines in your local built js files, there shouldn't be any ts issues with the changed lines.